polymer.externs.js

```diff
@@ -597,11 +604,11 @@ PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 
 /**
  * @param {!Function} callback
- * @return {!{fn: !Function=, _nodes: !Array<!Node>}}
+ * @return {!{fn: !Function, _nodes: !Array<!Node>}}
```

Comment:
Ah sorry this was my mistake in the comment on your previous CL. This should really be

`{!{fn: (!Function|undefined), _nodes: !Array<!Node>}}`

same below. Confusing, I know :-).